### ISSUE TITLE
Incomplete Documentation for Draper patch for Rails 5

### ISSUE THREAD

**Issue by juliusdelta** [owen2345/camaleon-cms#646]
It seems the docs at http://camaleon.tuzitio.com/documentation/category/40757-developer-docs/installation-existent-project.html are a little out of date.

Camaleon was installed on an existing Rails 5 app. It seems this rails commit [`8017e6a`](https://github.com/rails/rails/commit/8017e6af31caa58a58787274ff0ca01397219e49) removed requiring `Rake::TestTask`. See [this issue](https://github.com/rails/rails/issues/19997)

Basically
```ruby
class Rails::SubTestTask < Rake::TestTask
end
```
should to be updated to 
```ruby
require 'rake/testtask.rb'
class Rails::SubTestTask < Rake::TestTask
end
```

I'd submit a PR myself but at a quick peruse of the codebase I couldn't find the docs (they might be there.)

Just a heads up!

**Comment by brian-kephart** [comment]
That documentation also refers to a branch of the Draper gem that isn't necessary anymore (master is now compatible with Rails 5).

**Comment by juliusdelta** [comment]
Good point! If someone can point me to the docs I'll open a PR

**Comment by brian-kephart** [comment]
I think that website is separately maintained. I suggest closing this issue and contacting Owen [here](http://camaleon.tuzitio.com/#contact_section) instead.

**Comment by owen2345** [comment]
I updated the doc. Sorry for my delay but these days I am too busy.
Thank you guys for reporting!

**Comment by brian-kephart** [comment]
No problem @owen2345. Thanks for your dedication to this awesome project!
